conda/common/anaconda_client.py

```diff
@@ -119,23 +119,40 @@
 #     return re.sub(r'([./])conda([./])', r'\1api\2', url, count=1)
 
 
+def get_binstar_directory():
+    # returns an absolute directory path or None
+    try:
+        from binstar_client.utils.appdirs import AppDirs, EnvAppDirs
+    except ImportError:
+        return None
+
+    if 'BINSTAR_CONFIG_DIR' in os.environ:
+        return EnvAppDirs('binstar', 'ContinuumIO', os.environ['BINSTAR_CONFIG_DIR']).user_data_dir
+    else:
+        return AppDirs('binstar', 'ContinuumIO').user_data_dir
+
+
+def get_anaconda_default_site():
+    # returns site_name or None if not set
+    pass
+
+
+def get_anaconda_site_url(site_name):
+    # returns the url for an anaconda_site
+    pass
+
+
 def replace_first_api_with_conda(url):
     # replace first occurrence of 'api' with 'conda' in url
     return re.sub(r'([./])api([./])', r'\1conda\2', url, count=1)
 
 
 def read_binstar_tokens():
```

Comment:
TODO @srossross Need pointer to code that allows me to read currently logged in anaconda.org sites.  Here I'm pulling from the appdirs directory.  I'd rather just use the binstar_client logic for that.